README.md

```diff
@@ -352,29 +352,55 @@ To check if it's running well use `docker-compose logs -f`.
 
 ## JSON-RPC API
 
+Pathfinder supports both version `v0.1.0` and `v0.2.1` of the Starknet JSON-RPC specification.
+The `path` of the URL used to access the JSON-RPC server determines which version of the API is served:
+
+- the 0.1.0 API is exposed on the `/` and `/rpc/v0.1` paths
+- the 0.2.1-rc1 API is exposed on the `/rpc/v0.2` path
```

Comment:
As far as I can see back-ticks are used throughout the doc, `v` is a part of the tag too.

```suggestion
- the `v0.1.0` API is exposed on the `/` and `/rpc/v0.1` paths
- the `v0.2.1-rc1` API is exposed on the `/rpc/v0.2` path
```